packages/subgraph/src/api.ts

```diff
@@ -17,6 +17,7 @@ type PartialSubgraphApiContext = Partial<SubgraphApiContext>
 
 /**
  * TheGraph API client for CoW Protocol.
+ * * @deprecated the tool will be removed in the future
```

Comment:
We might, but not certain yet. I'd say:

```suggestion
 * * @deprecated not currently maintained, might removed in the future
```